packages/headless/src/controllers/folded-result-list/headless-folded-result-list.ts

```diff
@@ -192,3 +222,22 @@ function loadFoldingReducer(
   engine.addReducers({search, configuration, folding, query});
   return true;
 }
+
+function searchForResult(
+  results: FoldedCollection[] | FoldedResult[],
+  compareCb: (result: FoldedResult) => Boolean
```

Comment:
typically `boolean` is lowercase in Typescript:
```suggestion
  compareCb: (result: FoldedResult) => boolean
```